nix/shell.nix

```diff
@@ -0,0 +1,10 @@
+(import
```

Comment:
Keep nix stuff in a separate 'nix' branch. Separate from general configurations.